scripts/globals/items/cumulus_cell.lua

```diff
@@ -16,7 +16,7 @@ end;
 function onItemUse(target)
     local encumbrance = target:getStatusEffect(dsp.effect.ENCUMBRANCE_I)
     local power = encumbrance:getPower()
-    local newpower = bit.band(power, bit.bnot(0x0020))
+    local newpower = bit.band(power, bit.bnot(32))
```

Comment:
in the interest of less copy paste, if you see bit. anything assume we keep hex ^^